app/room/page.js

```
@@ -0,0 +1,270 @@
+        </div>
+      </header>
+      
+      <main className="flex-1 container mx-auto p-4 flex flex-col md:flex-row gap-4">
+        <div className="flex-1 flex flex-col gap-4">
+          <div className="relative bg-black rounded-lg overflow-hidden h-3/4">
+            <video 
+              ref={remoteVideoRef}
+              className="w-full h-full object-contain"
+              autoPlay 
+              playsInline
+            />
```

Biome (1.9.4)

[error] Provide a track for captions when using audio or video elements.

Captions support users with hearing-impairments. They should be a transcription or translation of the dialogue, sound effects, musical cues, and other relevant audio information.

(lint/a11y/useMediaCaption)